cli/azd/pkg/infra/provisioning/manager.go

```diff
@@ -122,39 +123,20 @@ func (m *Manager) Destroy(ctx context.Context, deployment *Deployment, options D
 
 // Plans the infrastructure provisioning and orchestrates interactive terminal operations
 func (m *Manager) plan(ctx context.Context) (*DeploymentPlan, error) {
-	var deploymentPlan *DeploymentPlan
 
-	err := m.runAction(
-		ctx,
-		"Planning infrastructure provisioning",
-		m.interactive,
-		func(ctx context.Context, spinner *spin.Spinner) error {
-			planningTask := m.provider.Plan(ctx)
-
-			go func() {
-				for progress := range planningTask.Progress() {
-					m.updateSpinnerTitle(spinner, progress.Message)
-				}
-			}()
-
-			go m.monitorInteraction(spinner, planningTask.Interactive())
-
-			result, err := planningTask.Await()
-			if err != nil {
-				return err
-			}
-
-			deploymentPlan = result
-
-			return nil
-		},
-	)
+	planningTask := m.provider.Plan(ctx)
+	go func() {
```

Comment:
Before the progress reporting was driving the UX drawing... It look like now we're just draining the channel and logging the messages?

Reply:
Yes, I remove the `runAction` wrapper that was orchestrating the spinner and the `monitorInteraction` while the action was running. 
I couldn't remove the `planningTask.Progress()` because there's a long path for it, from those components using the channel.  It's not relevant for the PR scope to remove that channel, so it is currently only making sure that the messages in the channel are received to avoid blocking a thread.